webpack.config.js

```diff
@@ -89,7 +113,7 @@ module.exports = (env, argv) => {
 
             // This fixes duplicate files showing up in chrome with sourcemaps enabled.
             // See https://github.com/webpack/webpack/issues/7128 for more info.
-            namedModules: false,
+            namedModules: true,
```

Comment:
If you're changing the value here, please update the comment too, otherwise this is going to be incredibly confusing. Is it OK to change this? Is the bug nor an issue any more? Why do we need to change it now?

Reply:
Thanks for catching! That's a leftover when I tried to make it work without `style-loader` and completely forgot about it.